### PR TITLE
Update settingUp.md #5131

### LINKED ISSUE #5131: SettingUp.md: recommend Source Tree instead of GitHub for Windows
The current SettingUp.md says `Install GitHub for Windows/Mac (recommended), or at least, Git`

Should be changed as `Install Source Tree or other similar Git Client, or at least Git`.

### PR DESCRIPTION
**Pull request by ZhaoDingfan**
Fixes #5131 

Basically just change the sentence in settingUp.md, commit and push it.

### CODE REVIEW

File: WebContent/WEB-INF/web.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (xpdavid) [inline WebContent/WEB-INF/web.xml]: This change is not relevant. Pls discard it.

File: docs/settingUp.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
-
```
Comment (xpdavid) [inline docs/settingUp.md]: Unnecessary changes?

**Comment by ZhaoDingfan** [comment]
Sorry I messed it up... Trying to fix it

**Comment by xpdavid** [comment]
@ZhaoDingfan You can always `rebase` and `squash` for PR.

**Comment by ZhaoDingfan** [comment]
@xpdavid  I will do it in a new pull request because I made some mistakes in current branch and don't know how to revert back..
